fcrepo-clients/src/test/java/edu/unc/lib/dl/event/RepositoryPremisLoggerIT.java

```diff
@@ -166,4 +169,70 @@ public void getEventsModelForObjectWithoutLog() throws Exception {
 
         assertTrue(eventsModel.isEmpty());
     }
+
+    @Test
+    public void makeMultipleChangesSimultaneously() throws InterruptedException {
+        parentObject = repoObjFactory.createDepositRecord(null);
+        initPremisLogger(parentObject);
+
+        // make sure that there are no events in premis log before the writes
+        PremisLogger retrieveLogger = new RepositoryPremisLogger(parentObject, mockSession,
+                pidMinter, repoObjLoader, repoObjFactory);
+        Model initialLogModel = retrieveLogger.getEventsModel();
+        assertFalse("New premis already contains events", initialLogModel.listObjects().hasNext());
+
+        // add original event
+        Resource originalEvent = logger.buildEvent(Premis.note)
```

Comment:
It would be more valuable to check that this event exists after the concurrent operations below take place, to show that the whole log makes it through. We can count on the other tests to already be making sure that normal addition of events works